types/ramda/index.d.ts

```diff
@@ -293,15 +294,34 @@ export function complement<TArgs extends any[]>(pred: (...args: TArgs) => unknow
  * functions must be unary.
  */
 // tslint:disable:max-line-length
+export function compose<TArgs extends any[], R1, R2, R3, R4, R5, R6, R7, TResult>(
```

Comment:
@Nicholaiii  I try to change it to multiple lines. what's your opinions of this style?

Reply:
I generally prefer 2-space indentation like you use, but the rest of the document uses four spaces. I also think putting the closing bracket and final return statement on same level as export, but you can see this in my example

Reply:
I think we should go for 4, for uniformity.

Reply:
> 

OK

Reply:
Done